kbase-extension/static/kbase/js/util/jobLogViewer.js

```diff
@@ -1045,7 +1235,10 @@ define([
         }
 
         function doJobNotFound() {
-            ui.setContent('log-panel', div([p(['No job found; logs cannot be displayed'])]));
+            // clear the log container
+            ui.setContent('log-container', '');
+            renderJobState({ job_state: 'does_not_exist' });
+            stopJobStatusUpdates();
```

Comment:
Obviously if the job is not found, there's no point in getting status updates on it.